### PR TITLE
[systeminfo][tv] add callback for change of source_info

### PR DESCRIPTION
**Pull request by djuszczakk**
[ACR]
http://suprem.sec.samsung.net/jira/browse/TWDAPI-235

Signed-off-by: Dawid Juszczak <d.juszczak@samsung.com>

### CODE REVIEW

**Comment by djuszczakk** [comment]
> Need to update system-information.md guide.

Updated

File: docs/application/web/guides/device/system-information.md [hunk 1]
```diff
@@ -282,6 +282,10 @@ The system properties are defined as subtypes of the `SystemInfoProperty` interf
...
 | `SystemInfoMemory` (in [mobile](../../api/latest/device_api/mobile/tizen/systeminfo.html#SystemInfoMemory), [wearable](../../api/latest/device_api/wearable/tizen/systeminfo.html#SystemInfoMemory), and [TV](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoMemory) applications) | `MEMORY`             | Provides information about the memory state of the device. |
 | `SystemInfoADS` (in [mobile](../../api/latest/device_api/mobile/tizen/systeminfo.html#SystemInfoADS), [wearable](../../api/latest/device_api/wearable/tizen/systeminfo.html#SystemInfoADS), and [TV](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoADS) applications) | `ADS`                | Provides information about the advertisement service. |
+| `SystemInfoVideoSource` (in [TV](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoVideoSource) applications) | `VIDEOSOURCE`                | Provides information about connected and disconnected devices. |
```
Comment (mijin85cho) [inline docs/application/web/guides/device/system-information.md]: If only one profile is supported, you don't need to use "(in ~~ application)" form.
Applying link directly as like [SystemInfoVideoSource](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoVideoSource) is ok.

**Comment by mijin85cho** [comment]
Hello, @surbhig12 
Would you please check updated part of  docs/application/web/guides/device/system-information.md only?
Thanks in advance.

File: docs/application/web/guides/device/system-information.md [hunk 2]
```diff
@@ -282,6 +282,10 @@ The system properties are defined as subtypes of the `SystemInfoProperty` interf
...
 | `SystemInfoMemory` (in [mobile](../../api/latest/device_api/mobile/tizen/systeminfo.html#SystemInfoMemory), [wearable](../../api/latest/device_api/wearable/tizen/systeminfo.html#SystemInfoMemory), and [TV](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoMemory) applications) | `MEMORY`             | Provides information about the memory state of the device. |
 | `SystemInfoADS` (in [mobile](../../api/latest/device_api/mobile/tizen/systeminfo.html#SystemInfoADS), [wearable](../../api/latest/device_api/wearable/tizen/systeminfo.html#SystemInfoADS), and [TV](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoADS) applications) | `ADS`                | Provides information about the advertisement service. |
+| [`SystemInfoVideoSource`](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoVideoSource) | `VIDEOSOURCE`                | Provides information about connected and disconnected devices. |
```
[the remark this answers is not in the document]
Comment (djuszczakk) [reply docs/application/web/guides/device/system-information.md]: Done.

File: docs/application/web/guides/device/system-information.md [hunk 3]
```diff
@@ -282,6 +282,10 @@ The system properties are defined as subtypes of the `SystemInfoProperty` interf
...
+| `SystemInfoCameraFlash` (in [mobile](../../api/latest/device_api/mobile/tizen/systeminfo.html#SystemInfoCameraFlash), [wearable](../../api/latest/device_api/wearable/tizen/systeminfo.html#SystemInfoCameraFlash), and [TV](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoCameraFlash) applications) | `CAMERA_FLASH`                | Provides the way to control the attached camera flash. |
```
[the remark this answers is not in the document]
Comment (djuszczakk) [reply docs/application/web/guides/device/system-information.md]: Done.

File: docs/application/web/guides/device/system-information.md [hunk 4]
```diff
@@ -282,6 +282,10 @@ The system properties are defined as subtypes of the `SystemInfoProperty` interf
...
+| [`SystemInfoServiceCountry`](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoServiceCountry) | `SERVICE_COUNTRY`                | Represents a country of which basic policy of terms and conditions is set. |
```
[the remark this answers is not in the document]
Comment (djuszczakk) [reply docs/application/web/guides/device/system-information.md]: Done.

File: docs/application/web/guides/device/system-information.md [hunk 5]
```diff
@@ -282,6 +282,10 @@ The system properties are defined as subtypes of the `SystemInfoProperty` interf
...
+| [`SystemInfoSourceInfo`](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoVideoSourceInfo) | `SOURCE_INFO`                | Provides information about current video source of device, eg. TV, HDMI1, HDMI2 etc. |
```
[the remark this answers is not in the document]
Comment (djuszczakk) [reply docs/application/web/guides/device/system-information.md]: Done.

**Comment by mijin85cho** [comment]
Hello, @surbhig12.
It looks like this PR is updated. Would you please check?

File: docs/application/web/guides/device/system-information.md [hunk 6]
```diff
@@ -282,6 +282,10 @@ The system properties are defined as subtypes of the `SystemInfoProperty` interf
...
+| [`SystemInfoVideoSource`](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoVideoSource) | `VIDEOSOURCE`                | Provides information about the connected and disconnected devices. |
+| `SystemInfoCameraFlash` (in [mobile](../../api/latest/device_api/mobile/tizen/systeminfo.html#SystemInfoCameraFlash), [wearable](../../api/latest/device_api/wearable/tizen/systeminfo.html#SystemInfoCameraFlash), and [TV](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoCameraFlash) applications) | `CAMERA_FLASH`                | Provides a way to control the attached camera flash. |
+| [`SystemInfoServiceCountry`](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoServiceCountry) | `SERVICE_COUNTRY`                | Represents a country for which the basic policy of terms and conditions is set. |
+| [`SystemInfoSourceInfo`](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoVideoSourceInfo) | `SOURCE_INFO`                | Provides information about the current video source of device, for example TV, HDMI1, HDMI2 and so on. |
```
Comment (surbhig12) [inline docs/application/web/guides/device/system-information.md]: Minor comment. Missing serial comma before and:

Provides information about the current video source of device, for example TV, HDMI1, HDMI2, and so on.

Comment (djuszczakk) [reply docs/application/web/guides/device/system-information.md]: Done.

**Pull request merged**